kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/DeltaTableFeaturesSuite.scala

```diff
@@ -284,6 +286,37 @@ class DeltaTableFeaturesSuite extends DeltaTableWriteSuiteBase {
     }
   }
 
+  test("UNIVERSAL_FORMAT feature can be populated") {
+    withTempDirAndEngine { (tablePath, engine) =>
+      val properties =
+        Map(
+          TableConfig.UNIVERSAL_FORMAT_ENABLED_FORMATS.getKey -> "iceberg",
+          TableConfig.ICEBERG_COMPAT_V2_ENABLED.getKey -> "true")
+      createEmptyTable(engine, tablePath, testSchema, tableProperties = properties)
+
+      val table = Table.forPath(engine, tablePath)
+      val writtenSnapshot = latestSnapshot(table, engine)
+      assert(TableConfig.UNIVERSAL_FORMAT_ENABLED_FORMATS.fromMetadata(
+        writtenSnapshot.getMetadata).contains(UniversalFormats.FORMAT_ICEBERG))
+    }
+  }
+
+  test("UNIVERSAL_FORMAT feature will throw if icebergCompatV2 was not enabled") {
+    withTempDirAndEngine { (tablePath, engine) =>
+      createEmptyTable(engine, tablePath, testSchema)
+
+      val table = Table.forPath(engine, tablePath)
+
+      intercept[InvalidConfigurationValueException] {
```

Comment:
Can you check the error message either here or in the unit test?

Reply:
checked in unit test.